src/content/peerconnection/webaudio-input/js/main.js

```diff
@@ -67,10 +67,10 @@ function handleSuccess(stream) {
     console.log('Got one audio track:', audioTracks);
     const filteredStream = webAudio.applyFilter(stream);
     const servers = null;
-    pc1 = new webkitRTCPeerConnection(servers); // eslint-disable-line new-cap
+    pc1 = new RTCPeerConnection(servers); // eslint-disable-line new-cap
```

Comment:
Since you're now using a caps constructor, I think you can remove the eslint thingy.